app/models/verification/validator/beds_and_plates_scanned.rb

```diff
@@ -1,31 +1,68 @@
 # frozen_string_literal: true
 
 class Verification::Validator::BedsAndPlatesScanned < ActiveModel::Validator
+  # Validates the scanned values of the verification record. It checks if the
+  # bed exists for the instrument and the bed barcode matches the one specified
+  # in the input.
+  #
+  # @param record [Verification::Base] the record to validate
+  # @return [void]
   def validate(record)
     record.scanned_values.each do |bed_name, position|
-      unless valid_bed?(bed_name, position[:bed], position[:plate], record)
-        record.errors.add(:base, "Invalid layout")
-        break
-      end
+      next if blank_position?(position)
```

Comment:
II'm not entirely sure how this feature is used, but it seems like we only need to skip blank positions without notifying the user about them. Just wanted to confirm if this is the intended behaviour.

Reply:
Thanks. There are 4 rows each with a source and destination positions, which are shown as boxes in the UI. If both source and destination are not specified for a row, that row is skipped because those are not used. That is the intended behaviour.

Reply:
Thanks for the confirmation 👍

Reply:
I checked this again. Blank position is skipped in this validator because it only validates the scanned values here but the matching pair is validated in another validator.

Reply:
Thanks for checking. Sounds good 👍